### ISSUE TITLE
Charts - provide an accessible title and description

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1498]
The charts currently are not accessible to screen reader users. The following css-tricks article describes how charts can be made accessible, by adding a title and description of the contents:
https://css-tricks.com/accessible-svgs/#article-header-id-6

The chart components we provide should include a way for consumers to provide a title and description, and have those strings included in the svg as described in the article.

**Comment by rachael-phillips** [comment]
Hi @dlabrecq, I wanted to make you aware of this issue. Would this be something you would be interested in incorporating in your work on charts?

**Comment by dlabrecq** [comment]
To say that the charts are not accessible is probably not a fair statement. Victory charts support properties for description and title today.

**Comment by rachael-phillips** [comment]
Hi @jgiardino do you think we need to provide the title and description in our examples or is it sufficient that victory charts supports properties for description and title?

**Comment by jgiardino** [comment]
Thanks for the links to the documentation, @dlabrecq! I had looked for similar options for the victory charts but couldn't find them, and assumed that wasn't even supported. So that's awesome that they are!  

I think this issue is still valid in that the argument can be made that our chart examples aren't accessible. I see the following as the two main action items for this issue:
- Verify that `desc` and `title` can be used in our implementation of victory charts. I only did a quick test using the inline editor on our website, but I wasn't able to figure out how to define those in our component.
- Update our chart examples to include `title` and `desc`. Ideally, our examples should represent best practice where possible, since many devs refer to our examples for ideas on how to implement them. These settings are likely to be overlooked if we don't include them. And it also makes it easier to figure out how to set them when they're already defined in our examples.

**Comment by dlabrecq** [comment]
We need to keep this open and add the desc and title properties to the chart examples.
